src/include/wiredtiger.h.in

```diff
@@ -5707,6 +5729,46 @@ struct __wt_page_log {
     int (*terminate)(WT_PAGE_LOG *page_log, WT_SESSION *session);
 };
 
+/*!
+ * The interface implemented by applications to provide a key management service
+ * implementation.
+ *
+ * Applications register implementations with WiredTiger by calling
+ * WT_CONNECTION::set_key_management.
+ *
+ * @snippet ex_key_management.c WT_KEY_MANAGEMENT register
+ */
+struct __wt_key_management {
+    /*!
+     * Fetch the current persisted key blob.
+     *
+     * @param km the WT_KEY_MANAGEMENT
+     * @param blob_data the blob data returned
+     * @param blob_size the size of the blob data returned
+     */
+    int (*get_key_blob)(WT_KEY_MANAGEMENT *km,
```

Comment:
I see that all extension API methods return as int. I am curious whether this is extension API rule or not? Let me know thanks